src/ragas/experimental/testset/transforms/extractors/llm_base.py

```diff
@@ -0,0 +1,160 @@
+    input_model: t.Type[StringIO] = StringIO
+    output_model: t.Type[StringIO] = StringIO
+    examples: t.List[t.Tuple[StringIO, StringIO]] = [
+        (
+            StringIO(
+                text="Artificial intelligence\n\nArtificial intelligence is transforming various industries by automating tasks that previously required human intelligence. From healthcare to finance, AI is being used to analyze vast amounts of data quickly and accurately. This technology is also driving innovations in areas like self-driving cars and personalized recommendations."
+            ),
+            StringIO(
+                text="AI is revolutionizing industries by automating tasks, analyzing data, and driving innovations like self-driving cars and personalized recommendations."
+            ),
+        )
+    ]
+
+
+class Keyphrases(BaseModel):
+    keyphrases: t.List[str]
+
+
+class KeyphrasesExtractorPrompt(PydanticPrompt[StringIO, Keyphrases]):
+    instruction: str = "Extract top 5 keyphrases from the given text."
+    input_model: t.Type[StringIO] = StringIO
+    output_model: t.Type[Keyphrases] = Keyphrases
+    examples: t.List[t.Tuple[StringIO, Keyphrases]] = [
+        (
+            StringIO(
+                text="Artificial intelligence\n\nArtificial intelligence is transforming various industries by automating tasks that previously required human intelligence. From healthcare to finance, AI is being used to analyze vast amounts of data quickly and accurately. This technology is also driving innovations in areas like self-driving cars and personalized recommendations."
+            ),
+            Keyphrases(
+                keyphrases=[
+                    "Artificial intelligence",
+                    "automating tasks",
+                    "healthcare",
+                    "self-driving cars",
+                    "personalized recommendations",
+                ]
+            ),
+        )
+    ]
+
+
+class TitleExtractorPrompt(PydanticPrompt[StringIO, StringIO]):
+    instruction: str = "Extract the title of the given document."
+    input_model: t.Type[StringIO] = StringIO
+    output_model: t.Type[StringIO] = StringIO
+    examples: t.List[t.Tuple[StringIO, StringIO]] = [
+        (
+            StringIO(
+                text="Deep Learning for Natural Language Processing\n\nAbstract\n\nDeep learning has revolutionized the field of natural language processing (NLP). This paper explores various deep learning models and their applications in NLP tasks such as language translation, sentiment analysis, and text generation. We discuss the advantages and limitations of different models, and provide a comprehensive overview of the current state of the art in NLP."
+            ),
+            StringIO(text="Deep Learning for Natural Language Processing"),
+        )
+    ]
+
+
+class Headlines(BaseModel):
+    headlines: t.Dict[str, t.List[str]]
+
+
+class HeadlinesExtractorPrompt(PydanticPrompt[StringIO, Headlines]):
```

Comment:
Can you replace the example here with a more generalizable and shorter one? this one targets Arxiv documents specifically.